__tests__/globals_test.ml

```diff
@@ -228,8 +228,9 @@ let () =
       );
     );
 
+(** run all timers to expiration but first, enable fake timers before manipulating them *)
     describe "timeout should not fail suite" (fun () ->
-      afterAllAsync ~timeout:1 (fun (finish) -> Jest.runAllTimers(); finish ());
+      Jest.useFakeTimers(); afterAllAsync ~timeout:1 (fun (finish) -> Jest.runAllTimers(); finish ());
```

Comment:
Seems we could also use `Jest.useFakeTimers("legacy")` to get the old behaviour back. Or set `"timers": "legacy"` in the config.

Reply:
Yes.  I'll update the PR.

Reply:
useFakeTimers now takes an optional implementation parameter
`
type fakeTimerImplementation = [ #legacy | #modern ]
`
`
let useFakeTimers = (~implementation: option<fakeTimerImplementation>=?, ())
`